clu/src/lib.rs

```diff
@@ -65,7 +63,7 @@ fn linux_generate_mesh_ip(config: &mut NetworkSettings) -> Result<(), Error> {
 }
 
 fn validate_wg_key(key: &str) -> bool {
-    key.len() == 44 && key.ends_with("=")
+    key.len() == 44 && key.ends_with("=") && !key.contains(" ")
```

Comment:
I wonder if this function wouldn't benefit from returning `Result`, because right now if something's wrong it won't tell the caller what.

Reply:
This applies to `validate_mesh_ip` too

Reply:
well lets think about what could go wrong, it has to be a string, rust will enforce that, it's a string all these functions will work unless there's a stdlib error which I don't think we should concern ourselves with catching at every corner case.

Reply:
because the function can only fail one way, I think its fine to use a bool